lib/upgrader.js

```diff
@@ -73,46 +86,42 @@ function replace({ argv }) {
     [ 'apos.areas.plaintext', 'apos.area.plaintext' ],
     [ 'apos.areas.walk', 'apos.area.walk' ]
   ];
-  for (const file of files) {
-    const original = fs.readFileSync(file, { encoding: 'utf8' });
-    let transformed = original;
-    for (const [ pattern, replacement ] of replaces) {
-      transformed = replaceAll(transformed, pattern, replacement);
-    }
-    if (original !== transformed) {
-      fs.writeFileSync(file, transformed);
-    }
-  }
+
+  replaceOccurences(files, replaces);
+}
+
+function renameModulesDeclarations(isMultisite) {
+  const files = [
+    'app.js',
+    ...isMultisite
+      ? [
+        'dashboard/index.js',
+        'sites/index.js',
+        ...globByPattern('dashboard/lib/theme-*.js'),
+        ...globByPattern('sites/lib/theme-*.js')
+      ]
+      : []
+  ];
+
+  const renamedReplaces = Object.entries(renamedModulesMap)
```

Comment:
removes the prefix used to distinguish between dashboard and sites in `renamedModulesMap` in order to match occurences in files.